### PR TITLE
Testing/single page

### LINKED ISSUE #66: Improve test coverage

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [ ] Feature
- [X] Refactor
- [ ] Style
- [X] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Refactor/add tests to reflect Single Page view updates

### What ticket(s) do the changes resolve?
close #66 
Please Link Issues to this PR as necessary.
#66 
### Notes for the Reviewer
All tests should be passing, system is checking for unique planet numbers.

### How to Test Changes?
switch to branch, run `npm start` and `npm run cypress` in separate tabs in terminal, go through all four testing suites to ensure all are passing.

### CODE REVIEW

File: cypress/integration/single-planet.js [hunk 1]
```diff
@@ -12,28 +12,57 @@ describe('Single Planet Testing', () => {
...
+  })
+  it('Should only allow seven numbers to be entered', () => {
+    cy.visit(`${baseURL}mars`)
+      .get('.gravity-words').should('contain', '38 pounds')
+    cy.get('input[name="weight"]').clear().type('111204')
+      .get('.gravity-words').should('contain', '42069 pounds')
+    cy.get('input[name="weight"]').clear().type('1111111111')
+      .get('.gravity-words').should('contain', '420335 pounds')
+    cy.get('input[name="weight"]').clear().type('1111111')
+      .get('.gravity-words').should('contain', '420335 pounds')
```
Comment (alexthompson207) [inline cypress/integration/single-planet.js:65]: very thorough testing coverage, way to go!

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 2]
```diff
@@ -89,6 +89,7 @@ const PlanetInfo: React.FC<InfoProps> = ({ currentPlanet, resetSort }): JSX.Elem
                 Something that weighs
                 <input
                   className='gravity-input'
+                  name='weight'
```
Comment (alexthompson207) [inline src/components/PlanetInfo/PlanetInfo.tsx:92]: Good catch here!

**Review by alexthompson207 — APPROVED** [review]
Really solid work with testing Connor!

File: src/components/PlanetInfo/PlanetInfo.tsx (hunk 2 above)
Comment (knees4bees) [reply src/components/PlanetInfo/PlanetInfo.tsx:92]: Yep, this will help with accessibility!

**Review by knees4bees — APPROVED** [review]
So many more tests! This is great.

**Pull request merged**